### ISSUE TITLE
 在新数据上的测试结果比测试集差很多  The test result on the new data is much worse than the test set

### ISSUE THREAD

**Issue by kangxi9** [AlexeyAB/darknet#8691]
Hello, I'm doing object detection for person, using single channel grayscale image. 
1. The model has achieved good results in both training set and test set, as shown in Figure 1. There is basically no missing or false detection in the test set containing 900 pictures. 
2. However, when testing on the data of 300 newly collected pictures, I was surprised by the results. The first dozens of pictures can detect people, but the location of the box is not very accurate and false alarm, as shown in Figure 2. 
3. The rest of the pictures have many disordered frames, as shown in Figure 3. I don't know why. Is it because the newly collected data is different from the training data or the model generalization ability is poor? I look forward to receiving your advice and reply.

![figure1](https://user-images.githubusercontent.com/65488146/198519759-cd8b5fbb-1b89-4d1e-a326-bc722edf0355.png)

![figure2](https://user-images.githubusercontent.com/65488146/198519870-7b7428d2-f30b-4b38-bbc0-bb7f347b7397.png)

![figure3](https://user-images.githubusercontent.com/65488146/198519921-c76489c1-af78-4c66-9c20-370568af7ff4.png)

**Comment by 1027663760** [comment]
Continue to increase the number of training set pictures
